clerk/tokens.go

```diff
@@ -2,50 +2,52 @@ package clerk
 
 import (
 	"fmt"
+	"strings"
 	"time"
 
 	"gopkg.in/square/go-jose.v2"
 
 	"gopkg.in/square/go-jose.v2/jwt"
 )
 
-type Claims struct {
+var (
+	standardClaimsKeys = []string{"iss", "sub", "aud", "exp", "nbf", "iat", "jti"}
+)
+
+type TokenClaims struct {
 	jwt.Claims
-	UserClaims
+	Extra map[string]interface{}
 }
 
-type UserClaims struct {
-	Name                string `json:"name"`
-	Picture             string `json:"picture"`
-	UpdatedAt           int64  `json:"updated_at"`
-	GivenName           string `json:"given_name,omitempty"`
-	FamilyName          string `json:"family_name,omitempty"`
-	PreferredUsername   string `json:"preferred_username,omitempty"`
-	Gender              string `json:"gender,omitempty"`
-	Birthdate           string `json:"birthdate,omitempty"`
-	Email               string `json:"email,omitempty"`
-	EmailVerified       bool   `json:"email_verified,omitempty"`
-	PhoneNumber         string `json:"phone_number,omitempty"`
-	PhoneNumberVerified bool   `json:"phone_number_verified,omitempty"`
+type SessionClaims struct {
+	jwt.Claims
```

Comment:
🔧 Why don't we specify explicitly only the ones that the session tokens contain?

Reply:
Will change it accordingly to only contain the session token claims

Reply:
Actually by not using the `jwt.Claims` we will have to implement the validation on our own and we won't leverage the package's helper method `ValidateWithLeeway`

Reply:
Cool, let's leave it as is then.